SalaryService.md

```diff
@@ -0,0 +1,104 @@
+Salary Service
+====================
+
+Table of Contents
+_____________
+
+- [Summary](#summary)
+- [Request structure](#request-structure)
+- [Response structure](#response-structure)
+- [Versioning](#versioning)
+
+Summary
+-----------
+The Salary service returns salary info for a carotene id and, optionally, a postal or a cbsa code in either hourly or yearly amounts.
```

Comment:
```suggestion
The Salary service returns salary info for a carotene id and, optionally, a postal or a CBSA code. Response data can be returned in either hourly or yearly amounts.
```

Reply:
It is maybe a bit ambiguous if your last prepositional phrase is modifying  'salary info', 'cbsa coda', 'postal code', etc.